### DISCUSSION TITLE
Gateway list - aws s3 cross account

### DISCUSSION THREAD

**Discussion by Fran-Rg** [minio/minio#16426] [Q&A] [answered]
Hi,

I know Gateway is deprecated now and I'm not expecting new features but I'm wondering if it's expected for the gateway to not be able to list and permit access to a bucket in another aws account? The role granted to minio service has access to the bucket for sure but once on the minio browser the bucket isn't listed and the "mc ls <alias>" returns the same.

Is there a workaround to force/allow that other bucket to be visible?

I found a similar unanswered question SO: https://stackoverflow.com/questions/64395358/minio-unable-to-access-bucket-in-another-aws-account

**Comment by harshavardhana** [comment, marked as answer]
Gateway is not only deprecated, it's removed from our software stack. This question is mostly invalid for us.
